### DISCUSSION TITLE
ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'annotationStorage' of null

### DISCUSSION THREAD

**Discussion by Pootzjava** [stephanrauh/ngx-extended-pdf-viewer#829] [General]
My name is Agnaldo 
I'm trying implement multiple pdfs that will show according with a choose in a select.
When I select a calling in server is done create the pdf in server and the url of the pdf created is retorned, the process works in the first time that the screen is loaded but when I select again, all process is redone but the pdf is not showed and happen the error down.  

thanks 

```
core.js:6456 ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'annotationStorage' of null
TypeError: Cannot read property 'annotationStorage' of null
    at NgxExtendedPdfViewerComponent.<anonymous> (ngx-extended-pdf-viewer.js:1897)
    at Generator.next (<anonymous>)
    at fulfilled (tslib.es6.js:73)
    at ZoneDelegate.invoke (zone.js:372)
    at Object.onInvoke (core.js:28677)
    at ZoneDelegate.invoke (zone.js:371)
    at Zone.run (zone.js:134)
    at zone.js:1276
    at ZoneDelegate.invokeTask (zone.js:406)
    at Object.onInvokeTask (core.js:28664)
    at resolvePromise (zone.js:1213)
    at zone.js:1120
    at rejected (tslib.es6.js:74)
    at ZoneDelegate.invoke (zone.js:372)
    at Object.onInvoke (core.js:28677)
    at ZoneDelegate.invoke (zone.js:371)
    at Zone.run (zone.js:134)
    at zone.js:1276
    at ZoneDelegate.invokeTask (zone.js:406)
    at Object.onInvokeTask (core.js:28664)
```

**Comment by stephanrauh** [comment]
Hi Agnaldo, nice meeting you!

I'm afraid I don't know which code is located at line 1897 of ngx-extended-pdf-viewer. There are so many version of ngx-extended-pdf-viewer out there. Would you mind to open the "source" tab of your browser, locate `NgxExtendedPdfViewerComponent:1897`, and send me a copy or screenshot of the source code you've found?

BTW, adding the attribute `[minifiedJSLibraries]="false"` makes my life a lot easier. Otherwise, you see the minified code, possibly mapped by a source map, but even with source maps it's not the same as the source code I wrote.

Thanks in advance,
Stephan

**Comment by stephanrauh** [comment]
Oh, hey, I just saw you're the first to open a thread in the discussion. Awesome! Everybody who has an idea how to solve this, please join us!

**Reply by Pootzjava** [reply]
Thanks by your attention
I got to Solve my problem. I used your showcase as example doing the needed changes, after three days looking for a solution; I only updated to version  12.1.1 the angular.

**Reply by stephanrauh** [reply]
That's a surprisingly simple solution. I can't evade the feeling there's a bug lurking in a dark corner. If you run into it again, or if you run into a different bug, don't hesitate to tell me!
